recipe/meta.yaml

```diff
@@ -8,24 +8,24 @@ source:
   sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
 
 build:
-  number: 0 
+  number: 1
   script: pip install --no-deps .
 
 requirements:
   build:
     - python
     - pip
     - zlib 1.2*
-    - libtiff 4*
-    - jpeg 8*
+    - libtiff 4.0.6
+    - jpeg 9*
     - freetype 2.5*
     - tk
   run:
     - python
-    - jpeg 8*
+    - jpeg 9*
     - zlib 1.2*
     - freetype 2.5*
-    - libtiff 4*
+    - libtiff 4.0.6
```

Comment:
> BTW I am re-building/pinning everything to xz 5.0.5 to avoid the Python 3.4/3.5 version conflict.

I suggest avoiding that - it is work that we will only need to undo eventually - we should try to find a long-term solution.